docs/set_up_svs.rst

```diff
@@ -12,14 +12,18 @@ drive or a DVD) and physically transfer the *Data Transfer Device* to
 the *Secure Viewing Station*.
 
 Since the *Secure Viewing Station* never uses a network connection or an
-internal hard drive, we recommend that you physically remove any any
-internal storage devices or networking hardware such as wireless cards
-or Bluetooth adapters. If the machine has network ports you can't
-physically remove, you should clearly cover these ports with labels
-noting not to use them. For an even safer approach, fill a port with
-epoxy to physically disable it. If you have questions about repurposing
-hardware for the *Secure Viewing Station*, contact the `Freedom of the
-Press Foundation <https://securedrop.org/help>`__.
+internal hard drive, we recommend that you physically remove any any internal
+storage devices or networking hardware such as wireless cards or Bluetooth
+adapters. If the machine has network ports you can't physically remove, you
+should clearly cover these ports with labels noting not to use them. For an even
+safer approach, fill a port with epoxy to physically disable it. We also
+recommend you remove the speakers from the device (or just cut the audio cables
+if that's easier) . This is to prevent exfiltration of data from the airgap via
+ultrasonic (i.e., above the range of human hearing) waves (for more read
+https://arstechnica.com/security/2013/12/scientist-developed-malware-covertly-jumps-air-gaps-using-inaudible-sound/).
```

Comment:
> This is to prevent **exfiltration of data from the airgap via ultrasonic audio**, which cannot be heard by humans. 

The bold part should be the link.